ResearchKit/Common/ORKPasscodeStepViewController.m

```diff
@@ -39,9 +39,14 @@ OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
 #import <AudioToolbox/AudioToolbox.h>
 #import <LocalAuthentication/LocalAuthentication.h>
 
+static CGFloat const kForgotPasscodeVerticalPadding     = 50.0f;
```

Comment:
Oooh, it gets better. If you dig in, they are also using constraints in the super class. Fun, fun!